src/engine/configuration.cpp

```diff
@@ -283,6 +304,11 @@ namespace {
         // multiprojector setups where a launcher window would be undesired
         std::optional<bool> bypassLauncher;
 
+        // Set which layer server should be preferd to be used, the options are
+        // \"All\", \"NewYork\", \"Sweden\" and \"Utah\".
+        std::optional<std::string> preferedLayerServer [[codegen::inlist("All", "NewYork",
```

Comment:
Alex and I talked about this and think this can be resolved by adding the option "None". That gives the user complete freedom to add any layer they want, we will just include the layers that do not depend on any single server and they can add anything they want on top of that.